Gruntfile.js

```diff
@@ -5,19 +5,19 @@ module.exports = function (grunt) {
     grunt.loadNpmTasks('grunt-execute')
     grunt.loadNpmTasks('grunt-karma')
 
-    let browser = grunt.option('browser')
-    let buildType = grunt.option('type')
+    const browser = grunt.option('browser')
```

Comment:
some simple linting fixes in this file to stop visual studio code from shouting at me